atomkraft/config/atomkraft_config.py

```diff
@@ -0,0 +1,25 @@
+import os
+from typing import Optional
+
+from atomkraft.config.config_file import ConfigFile
+from atomkraft.utils.project import project_root
+
+ATOMKRAFT_INTERNAL_DIR = ".atomkraft"
+
+# the file that contains internal configurations for atomkraft
+ATOMKRAFT_INTERNAL_CONFIG = "config.toml"
+
+
+class AtomkraftConfig(ConfigFile):
+    def __init__(self, path: Optional[str] = None):
+        if not path:
+            if "PYTEST_CURRENT_TEST" in os.environ:
+                root = "tests/project"
```

Comment:
I think I saw this somewhere else too. I don't like to add test code as part of original code.

This should be done via some env variable `PROJECT_ROOT` which overrides `project_root()` in runtime.

And when we are running `pytest`, we set `PROJECT_ROOT` in environment variables.